charts/argo-rollouts/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v1.0.2"
 description: A Helm chart for Argo Rollouts
 name: argo-rollouts
-version: 1.0.4
+version: 2.0.0
```

Comment:
Please update changelog on L14-15 with something like
```diff
   artifacthub.io/changes: |
-     - "[Fixed]: Add missing liveness and readiness probes"
-     - "[Changed]: Fix icon url"
+     - "[Added]: Dashboard manifests"
```

Reply:
Hi, @mkilchhofer , just want to confirm with you that the following lines need to be removed

```
-     - "[Fixed]: Add missing liveness and readiness probes"
-     - "[Changed]: Fix icon url"
```

Reply:
Yup, they need to be removed.

Reply:
Got it. Thanks.